tests/frontend/show.py

```diff
@@ -438,8 +438,9 @@ def setup_test():
         ("%{deps}", "- import-dev.bst\n- import-links.bst\n- import-bin.bst"),
         ("%{build-deps}", "- import-dev.bst\n- import-links.bst"),
         ("%{runtime-deps}", "- import-links.bst\n- import-bin.bst"),
+        ("%{artifact-cas-digest}", "(no artifact CAS digest)"),
```

Comment:
This now prints nothing.

```
E           AssertionError: Expected output:
E           (no artifact CAS digest)
E           Instead received output:
```

but maybe this whole change was supposed to be dropped now that we have a new file for these tests?

Reply:
Woops indeed, I only tested the new test and forgot to remove this, thanks.

Reply:
I believe this is fixed, closing.